diff -Naur x265_2.1.old/source/test/CMakeLists.txt x265_2.1/source/test/CMakeLists.txt
--- x265_2.1.old/source/test/CMakeLists.txt	2016-10-02 18:03:48.977201833 +0200
+++ x265_2.1/source/test/CMakeLists.txt	2016-10-02 18:05:54.580392467 +0200
@@ -44,7 +44,7 @@
     ipfilterharness.cpp ipfilterharness.h
     intrapredharness.cpp intrapredharness.h)
 
-target_link_libraries(TestBench x265-static ${PLATFORM_LIBS})
+target_link_libraries(TestBench x265-shared ${PLATFORM_LIBS})
 if(LINKER_OPTIONS)
     if(EXTRA_LIB)
         list(APPEND LINKER_OPTIONS "-L..")
