diff -Naur x265-4.3.old/source/dynamicHDR10/json11/json11.cpp x265-4.3/source/dynamicHDR10/json11/json11.cpp
--- x265-4.3.old/source/dynamicHDR10/json11/json11.cpp	2026-09-11 12:14:22.178154551 +0200
+++ x265-4.3/source/dynamicHDR10/json11/json11.cpp	2026-09-11 12:14:22.291068518 +0200
@@ -25,6 +25,7 @@
 #include <cstdlib>
 #include <cstdio>
 #include <limits>
+#include <cstdint>
 
 #if _MSC_VER
 #pragma warning(disable: 4510) //const member cannot be default initialized
