diff -Naur x265-4.3.old/source/CMakeLists.txt x265-4.3/source/CMakeLists.txt
--- x265-4.3.old/source/CMakeLists.txt	2026-09-11 12:14:22.134195812 +0200
+++ x265-4.3/source/CMakeLists.txt	2026-09-11 12:14:22.194689596 +0200
@@ -1158,7 +1158,15 @@
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
-        set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        if(HIGH_BIT_DEPTH)
+            if(MAIN12)
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main12)
+            else()
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main10)
+            endif()
+        else()
+            set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        endif(HIGH_BIT_DEPTH)
     endif()
     if(UNIX)
         set_target_properties(x265-shared PROPERTIES VERSION ${X265_BUILD})
diff -Naur x265-4.3.old/source/encoder/api.cpp x265-4.3/source/encoder/api.cpp
--- x265-4.3.old/source/encoder/api.cpp	2026-09-11 12:14:22.178887593 +0200
+++ x265-4.3/source/encoder/api.cpp	2026-09-11 12:14:22.194986336 +0200
@@ -1119,7 +1119,7 @@
 #define ext ".dylib"
 #else
 #include <dlfcn.h>
-#define ext ".so"
+#define ext ".so." xstr(X265_BUILD)
 #endif
 #if defined(__GNUC__) && __GNUC__ >= 8
 #pragma GCC diagnostic ignored "-Wcast-function-type"
