diff -Naur x265_4.1.old/source/common/aarch64/asm-primitives.cpp x265_4.1/source/common/aarch64/asm-primitives.cpp
--- x265_4.1.old/source/common/aarch64/asm-primitives.cpp	2024-12-03 12:10:55.403249774 +0100
+++ x265_4.1/source/common/aarch64/asm-primitives.cpp	2024-12-03 12:10:55.545250622 +0100
@@ -31,6 +31,7 @@
 
 extern "C" {
 #include "fun-decls.h"
+#include "../arm/ipfilter8.h"
 }
 
 #define ALL_LUMA_TU_TYPED(prim, fncdef, fname, cpu) \
