diff -Naur x265_3.4.old/source/CMakeLists.txt x265_3.4/source/CMakeLists.txt
--- x265_3.4.old/source/CMakeLists.txt	2020-06-23 21:24:07.628960428 +0200
+++ x265_3.4/source/CMakeLists.txt	2020-06-23 21:24:15.239219846 +0200
@@ -656,7 +656,15 @@
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
-        set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        if(HIGH_BIT_DEPTH)
+            if(MAIN12)
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main12)
+            else()
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main10)
+            endif()
+        else()
+            set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        endif(HIGH_BIT_DEPTH)
     endif()
     if(UNIX)
         set_target_properties(x265-shared PROPERTIES VERSION ${X265_BUILD})
diff -Naur x265_3.4.old/source/encoder/api.cpp x265_3.4/source/encoder/api.cpp
--- x265_3.4.old/source/encoder/api.cpp	2020-06-23 21:23:30.739702963 +0200
+++ x265_3.4/source/encoder/api.cpp	2020-06-23 21:24:15.239219846 +0200
@@ -1089,7 +1089,7 @@
 #define ext ".dylib"
 #else
 #include <dlfcn.h>
-#define ext ".so"
+#define ext ".so." xstr(X265_BUILD)
 #endif
 #if defined(__GNUC__) && __GNUC__ >= 8
 #pragma GCC diagnostic ignored "-Wcast-function-type"
