diff -Naur x265_4.0.old/source/common/aarch64/asm-primitives.cpp x265_4.0/source/common/aarch64/asm-primitives.cpp
--- x265_4.0.old/source/common/aarch64/asm-primitives.cpp	2024-10-01 10:53:48.838239452 +0200
+++ x265_4.0/source/common/aarch64/asm-primitives.cpp	2024-10-01 10:53:48.989240632 +0200
@@ -31,6 +31,7 @@
 
 extern "C" {
 #include "fun-decls.h"
+#include "../arm/ipfilter8.h"
 }
 
 #define ALL_LUMA_TU_TYPED(prim, fncdef, fname, cpu) \
