diff -Naur x265-4.3.old/source/common/aarch64/asm-primitives.cpp x265-4.3/source/common/aarch64/asm-primitives.cpp
--- x265-4.3.old/source/common/aarch64/asm-primitives.cpp	2026-09-11 12:14:22.136061449 +0200
+++ x265-4.3/source/common/aarch64/asm-primitives.cpp	2026-09-11 12:14:22.270298879 +0200
@@ -31,6 +31,7 @@
 
 extern "C" {
 #include "fun-decls.h"
+#include "../arm/ipfilter8.h"
 }
 
 #define ALL_LUMA_TU_TYPED(prim, fncdef, fname, cpu) \
