diff -Naur x265_4.1.old/source/CMakeLists.txt x265_4.1/source/CMakeLists.txt
--- x265_4.1.old/source/CMakeLists.txt	2024-12-03 15:55:35.383080967 +0100
+++ x265_4.1/source/CMakeLists.txt	2024-12-03 15:54:57.381845250 +0100
@@ -151,6 +151,7 @@
     option(ENABLE_LIBVMAF "Enable VMAF" OFF)
     if(ENABLE_LIBVMAF)
         add_definitions(-DENABLE_LIBVMAF)
+        include_directories(${VMAF_INCLUDE_DIR})
     endif()
 endif(UNIX)
 
@@ -890,10 +891,13 @@
     if(EXTRA_LIB)
         target_link_libraries(x265-shared ${EXTRA_LIB})
     endif()
-	 target_link_libraries(x265-shared ${PLATFORM_LIBS})
+        target_link_libraries(x265-shared ${PLATFORM_LIBS})
     if(SVTHEVC_FOUND)
         target_link_libraries(x265-shared ${SVT_HEVC_LIBRARY})
     endif(SVTHEVC_FOUND)
+    if(ENABLE_LIBVMAF)
+        target_link_libraries(x265-shared ${VMAF})
+    endif(ENABLE_LIBVMAF)
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
