diff -Naur x265_2.5.old/source/CMakeLists.txt x265_2.5/source/CMakeLists.txt
--- x265_2.5.old/source/CMakeLists.txt	2017-08-22 15:52:23.306377617 +0200
+++ x265_2.5/source/CMakeLists.txt	2017-08-22 15:52:43.531754785 +0200
@@ -581,7 +581,15 @@
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
-        set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        if(HIGH_BIT_DEPTH)
+            if(MAIN12)
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main12)
+            else()
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main10)
+            endif()
+        else()
+            set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        endif(HIGH_BIT_DEPTH)
     endif()
     if(UNIX)
         set_target_properties(x265-shared PROPERTIES VERSION ${X265_BUILD})
diff -Naur x265_2.5.old/source/encoder/api.cpp x265_2.5/source/encoder/api.cpp
--- x265_2.5.old/source/encoder/api.cpp	2017-08-22 15:51:27.387334826 +0200
+++ x265_2.5/source/encoder/api.cpp	2017-08-22 15:52:43.531754785 +0200
@@ -419,7 +419,7 @@
 #define ext ".dylib"
 #else
 #include <dlfcn.h>
-#define ext ".so"
+#define ext ".so." xstr(X265_BUILD)
 #endif
 
 static int g_recursion /* = 0 */;
