diff -Naur x265_2.7.old/source/CMakeLists.txt x265_2.7/source/CMakeLists.txt
--- x265_2.7.old/source/CMakeLists.txt	2018-04-24 21:44:50.859816691 +0200
+++ x265_2.7/source/CMakeLists.txt	2018-04-24 21:44:57.527939350 +0200
@@ -601,7 +601,15 @@
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
-        set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        if(HIGH_BIT_DEPTH)
+            if(MAIN12)
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main12)
+            else()
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main10)
+            endif()
+        else()
+            set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        endif(HIGH_BIT_DEPTH)
     endif()
     if(UNIX)
         set_target_properties(x265-shared PROPERTIES VERSION ${X265_BUILD})
diff -Naur x265_2.7.old/source/encoder/api.cpp x265_2.7/source/encoder/api.cpp
--- x265_2.7.old/source/encoder/api.cpp	2018-04-24 21:40:17.655791157 +0200
+++ x265_2.7/source/encoder/api.cpp	2018-04-24 21:44:57.528939368 +0200
@@ -473,7 +473,7 @@
 #define ext ".dylib"
 #else
 #include <dlfcn.h>
-#define ext ".so"
+#define ext ".so." xstr(X265_BUILD)
 #endif
 
 static int g_recursion /* = 0 */;
