diff -Naur x265_2.5.old/source/CMakeLists.txt x265_2.5/source/CMakeLists.txt
--- x265_2.5.old/source/CMakeLists.txt	2017-08-22 15:53:25.838543732 +0200
+++ x265_2.5/source/CMakeLists.txt	2017-08-22 15:53:42.724858632 +0200
@@ -70,9 +70,14 @@
     else()
         set(CROSS_COMPILE_ARM 0)
     endif()
-    message(STATUS "Detected ARM target processor")
     set(ARM 1)
-    add_definitions(-DX265_ARCH_ARM=1 -DHAVE_ARMV6=1)
+    if(${SYSPROC} STREQUAL "armv6l")
+        message(STATUS "Detected ARMv6 target processor")
+        add_definitions(-DX265_ARCH_ARM=1 -DHAVE_ARMV6=1)
+    elseif(${SYSPROC} STREQUAL "armv7l")
+        message(STATUS "Detected ARMv7 system processor")
+        add_definitions(-DX265_ARCH_ARM=1 -DHAVE_ARMV7=1)
+    endif()
 else()
     message(STATUS "CMAKE_SYSTEM_PROCESSOR value `${CMAKE_SYSTEM_PROCESSOR}` is unknown")
     message(STATUS "Please add this value near ${CMAKE_CURRENT_LIST_FILE}:${CMAKE_CURRENT_LIST_LINE}")
