diff -Naur x265_2.1.old/source/CMakeLists.txt x265_2.1/source/CMakeLists.txt
--- x265_2.1.old/source/CMakeLists.txt	2016-10-02 18:03:48.861199810 +0200
+++ x265_2.1/source/CMakeLists.txt	2016-10-02 18:03:58.374365728 +0200
@@ -514,7 +514,15 @@
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
-        set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        if(HIGH_BIT_DEPTH)
+            if(MAIN12)
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main12)
+            else()
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main10)
+            endif()
+        else()
+            set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        endif(HIGH_BIT_DEPTH)
     endif()
     if(UNIX)
         set_target_properties(x265-shared PROPERTIES VERSION ${X265_BUILD})
