diff -Naur x265-4.3.old/source/CMakeLists.txt x265-4.3/source/CMakeLists.txt
--- x265-4.3.old/source/CMakeLists.txt	2026-09-11 12:14:22.246514687 +0200
+++ x265-4.3/source/CMakeLists.txt	2026-09-11 12:14:22.248691301 +0200
@@ -195,6 +195,7 @@
     option(ENABLE_LIBVMAF "Enable VMAF" OFF)
     if(ENABLE_LIBVMAF)
         add_definitions(-DENABLE_LIBVMAF)
+        include_directories(${VMAF_INCLUDE_DIR})
     endif()
     if(X64)
         option(ENABLE_CET "Enable Control-flow Enforcement" OFF)
@@ -1151,10 +1152,13 @@
     if(EXTRA_LIB)
         target_link_libraries(x265-shared ${EXTRA_LIB})
     endif()
-	 target_link_libraries(x265-shared ${PLATFORM_LIBS})
+        target_link_libraries(x265-shared ${PLATFORM_LIBS})
     if(SVTHEVC_FOUND)
         target_link_libraries(x265-shared ${SVT_HEVC_LIBRARY})
     endif(SVTHEVC_FOUND)
+    if(ENABLE_LIBVMAF)
+        target_link_libraries(x265-shared ${VMAF})
+    endif(ENABLE_LIBVMAF)
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
