diff -Naur x265_3.5.old/source/cmake/Findsvthevc.cmake x265_3.5/source/cmake/Findsvthevc.cmake
--- x265_3.5.old/source/cmake/Findsvthevc.cmake	2021-03-17 16:53:52.670339577 +0100
+++ x265_3.5/source/cmake/Findsvthevc.cmake	2021-03-17 16:54:40.429142730 +0100
@@ -38,13 +38,7 @@
         string(REGEX MATCH "SVT_VERSION_PATCHLEVEL  \\(([0-9]*)\\)" _ ${version})
         set(SVT_VERSION_PATCHLEVEL ${CMAKE_MATCH_1})
 
-        if(NOT ${SVT_VERSION_MAJOR} EQUAL "1" OR NOT ${SVT_VERSION_MINOR} EQUAL "4" OR NOT ${SVT_VERSION_PATCHLEVEL} EQUAL "1")
-            message (SEND_ERROR "-- Found SVT-HEVC Lib Version: ${SVT_VERSION_MAJOR}.${SVT_VERSION_MINOR}.${SVT_VERSION_PATCHLEVEL} which doesn't match the required version: ${SVT_VERSION_MAJOR_REQUIRED}.${SVT_VERSION_MINOR_REQUIRED}.${SVT_VERSION_PATCHLEVEL_REQUIRED}; Aborting configure  ")
-        else()
-            message(STATUS "-- Found SVT-HEVC Lib Version: ${SVT_VERSION_MAJOR}.${SVT_VERSION_MINOR}.${SVT_VERSION_PATCHLEVEL}")
-        endif()
-    else()
-        message (SEND_ERROR "-- Required version of SVT-HEVC Lib: ${SVT_VERSION_MAJOR_REQUIRED}.${SVT_VERSION_MINOR_REQUIRED}.${SVT_VERSION_PATCHLEVEL_REQUIRED}; Aborting configure  ")
+        message(STATUS "-- Found SVT-HEVC Lib Version: ${SVT_VERSION_MAJOR}.${SVT_VERSION_MINOR}.${SVT_VERSION_PATCHLEVEL}")
     endif()
 endif()
 
diff -Naur x265_3.5.old/source/CMakeLists.txt x265_3.5/source/CMakeLists.txt
--- x265_3.5.old/source/CMakeLists.txt	2021-03-17 16:53:52.668339543 +0100
+++ x265_3.5/source/CMakeLists.txt	2021-03-18 09:33:35.818689816 +0100
@@ -592,12 +592,6 @@
         ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
 endif()
 
-if(SVTHEVC_FOUND)
-    install(FILES "${SVT_HEVC_INCLUDE_DIR}/EbApi.h" DESTINATION include)
-    install(FILES "${SVT_HEVC_INCLUDE_DIR}/EbErrorCodes.h" DESTINATION include)
-    install(FILES "${SVT_HEVC_LIBRARY}" DESTINATION ${LIB_INSTALL_DIR})
-endif()
-
 install(FILES x265.h "${PROJECT_BINARY_DIR}/x265_config.h" DESTINATION include)
 if((WIN32 AND ENABLE_CLI) OR (WIN32 AND ENABLE_SHARED))
     if(MSVC_IDE)
diff -Naur x265_3.5.old/source/encoder/api.cpp x265_3.5/source/encoder/api.cpp
--- x265_3.5.old/source/encoder/api.cpp	2021-03-17 16:53:52.938344084 +0100
+++ x265_3.5/source/encoder/api.cpp	2021-03-18 09:10:57.438837886 +0100
@@ -450,14 +450,18 @@
 
                 if (pic_in->rpu.payloadSize)
                 {
+#if ! SVT_CHECK_VERSION(1, 5, 0)
                     inputData->dolbyVisionRpu.payload = X265_MALLOC(uint8_t, 1024);
+#endif
                     memcpy(inputData->dolbyVisionRpu.payload, pic_in->rpu.payload, pic_in->rpu.payloadSize);
                     inputData->dolbyVisionRpu.payloadSize = pic_in->rpu.payloadSize;
                     inputData->dolbyVisionRpu.payloadType = NAL_UNIT_UNSPECIFIED;
                 }
                 else
                 {
+#if ! SVT_CHECK_VERSION(1, 5, 0)
                     inputData->dolbyVisionRpu.payload = NULL;
+#endif
                     inputData->dolbyVisionRpu.payloadSize = 0;
                 }
 
@@ -695,7 +699,9 @@
 
             svt_print_summary(enc);
             EB_H265_ENC_INPUT *inputData = (EB_H265_ENC_INPUT*)encoder->m_svtAppData->inputPictureBuffer->pBuffer;
+#if ! SVT_CHECK_VERSION(1, 5, 0)
             if (inputData->dolbyVisionRpu.payload) X265_FREE(inputData->dolbyVisionRpu.payload);
+#endif
 
             X265_FREE(inputData);
             X265_FREE(encoder->m_svtAppData->inputPictureBuffer);
@@ -2051,7 +2057,9 @@
     inputPtr->pBuffer = (unsigned char*)x265_malloc(sizeof(EB_H265_ENC_INPUT));
 
     EB_H265_ENC_INPUT *inputData = (EB_H265_ENC_INPUT*)inputPtr->pBuffer;
+#if ! SVT_CHECK_VERSION(1, 5, 0)
     inputData->dolbyVisionRpu.payload = NULL;
+#endif
     inputData->dolbyVisionRpu.payloadSize = 0;
 
 
