diff -Naur multicoreware-x265_git-931178347b3f.old/source/test/testharness.h multicoreware-x265_git-931178347b3f/source/test/testharness.h
--- multicoreware-x265_git-931178347b3f.old/source/test/testharness.h	2022-09-16 09:56:11.570907287 +0200
+++ multicoreware-x265_git-931178347b3f/source/test/testharness.h	2022-09-16 10:01:25.565913942 +0200
@@ -71,7 +71,7 @@
 #include <intrin.h>
 #elif (!defined(__APPLE__) && (defined (__GNUC__) && (defined(__x86_64__) || defined(__i386__))))
 #include <x86intrin.h>
-#elif ( !defined(__APPLE__) && defined (__GNUC__) && defined(__ARM_NEON__))
+#elif ( !defined(__APPLE__) && defined(__ARM_NEON__))
 #include <arm_neon.h>
 #else
 /* fallback for older GCC/MinGW */
