diff -Naur multicoreware-x265_git-8ee01d45b05c.old/source/CMakeLists.txt multicoreware-x265_git-8ee01d45b05c/source/CMakeLists.txt
--- multicoreware-x265_git-8ee01d45b05c.old/source/CMakeLists.txt	2023-09-30 09:16:35.921751270 +0200
+++ multicoreware-x265_git-8ee01d45b05c/source/CMakeLists.txt	2023-09-30 09:31:48.681228545 +0200
@@ -130,6 +130,7 @@
     option(ENABLE_LIBVMAF "Enable VMAF" OFF)
     if(ENABLE_LIBVMAF)
         add_definitions(-DENABLE_LIBVMAF)
+        include_directories(${VMAF_INCLUDE_DIR})
     endif()
 endif(UNIX)
 
@@ -723,10 +724,13 @@
     if(EXTRA_LIB)
         target_link_libraries(x265-shared ${EXTRA_LIB})
     endif()
-	 target_link_libraries(x265-shared ${PLATFORM_LIBS})
+        target_link_libraries(x265-shared ${PLATFORM_LIBS})
     if(SVTHEVC_FOUND)
         target_link_libraries(x265-shared ${SVT_HEVC_LIBRARY})
     endif(SVTHEVC_FOUND)
+    if(ENABLE_LIBVMAF)
+        target_link_libraries(x265-shared ${VMAF})
+    endif(ENABLE_LIBVMAF)
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
