diff -Naur x265_3.5.old/source/CMakeLists.txt x265_3.5/source/CMakeLists.txt
--- x265_3.5.old/source/CMakeLists.txt	2021-03-24 13:49:05.443220865 +0100
+++ x265_3.5/source/CMakeLists.txt	2021-03-24 16:45:18.434689953 +0100
@@ -121,6 +121,7 @@
     option(ENABLE_LIBVMAF "Enable VMAF" OFF)
     if(ENABLE_LIBVMAF)
         add_definitions(-DENABLE_LIBVMAF)
+        include_directories(${VMAF_INCLUDE_DIR})
     endif()
 endif(UNIX)
 
@@ -643,10 +644,13 @@
     if(EXTRA_LIB)
         target_link_libraries(x265-shared ${EXTRA_LIB})
     endif()
-	 target_link_libraries(x265-shared ${PLATFORM_LIBS})
+        target_link_libraries(x265-shared ${PLATFORM_LIBS})
     if(SVTHEVC_FOUND)
         target_link_libraries(x265-shared ${SVT_HEVC_LIBRARY})
     endif(SVTHEVC_FOUND)
+    if(ENABLE_LIBVMAF)
+        target_link_libraries(x265-shared ${VMAF})
+    endif(ENABLE_LIBVMAF)
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
