diff -Naur multicoreware-x265_git-8ee01d45b05c.old/source/CMakeLists.txt multicoreware-x265_git-8ee01d45b05c/source/CMakeLists.txt
--- multicoreware-x265_git-8ee01d45b05c.old/source/CMakeLists.txt	2023-09-30 09:13:19.163201633 +0200
+++ multicoreware-x265_git-8ee01d45b05c/source/CMakeLists.txt	2023-09-30 09:13:59.000919783 +0200
@@ -736,7 +736,15 @@
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
-        set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        if(HIGH_BIT_DEPTH)
+            if(MAIN12)
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main12)
+            else()
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main10)
+            endif()
+        else()
+            set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        endif(HIGH_BIT_DEPTH)
     endif()
     if(UNIX)
         set_target_properties(x265-shared PROPERTIES VERSION ${X265_BUILD})
diff -Naur multicoreware-x265_git-8ee01d45b05c.old/source/encoder/api.cpp multicoreware-x265_git-8ee01d45b05c/source/encoder/api.cpp
--- multicoreware-x265_git-8ee01d45b05c.old/source/encoder/api.cpp	2023-09-30 09:13:19.189202101 +0200
+++ multicoreware-x265_git-8ee01d45b05c/source/encoder/api.cpp	2023-09-30 09:13:59.002919820 +0200
@@ -1097,7 +1097,7 @@
 #define ext ".dylib"
 #else
 #include <dlfcn.h>
-#define ext ".so"
+#define ext ".so." xstr(X265_BUILD)
 #endif
 #if defined(__GNUC__) && __GNUC__ >= 8
 #pragma GCC diagnostic ignored "-Wcast-function-type"
