diff -Naur multicoreware-x265_git-931178347b3f.old/source/CMakeLists.txt multicoreware-x265_git-931178347b3f/source/CMakeLists.txt
--- multicoreware-x265_git-931178347b3f.old/source/CMakeLists.txt	2022-09-16 09:56:11.202900247 +0200
+++ multicoreware-x265_git-931178347b3f/source/CMakeLists.txt	2022-09-16 09:56:31.441287403 +0200
@@ -229,7 +229,7 @@
         add_definitions(-std=gnu++98)
     endif()
     if(ENABLE_PIC)
-         add_definitions(-fPIC)
+         add_definitions(-fPIC -DPIC)
     endif(ENABLE_PIC)
     if(NATIVE_BUILD)
         if(INTEL_CXX)
