diff -Naur x265_3.0.old/source/CMakeLists.txt x265_3.0/source/CMakeLists.txt
--- x265_3.0.old/source/CMakeLists.txt	2019-02-26 22:30:59.356613077 +0100
+++ x265_3.0/source/CMakeLists.txt	2019-02-26 22:31:07.500763202 +0100
@@ -611,7 +611,15 @@
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
-        set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        if(HIGH_BIT_DEPTH)
+            if(MAIN12)
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main12)
+            else()
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main10)
+            endif()
+        else()
+            set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        endif(HIGH_BIT_DEPTH)
     endif()
     if(UNIX)
         set_target_properties(x265-shared PROPERTIES VERSION ${X265_BUILD})
diff -Naur x265_3.0.old/source/encoder/api.cpp x265_3.0/source/encoder/api.cpp
--- x265_3.0.old/source/encoder/api.cpp	2019-02-26 22:29:35.521067694 +0100
+++ x265_3.0/source/encoder/api.cpp	2019-02-26 22:31:07.500763202 +0100
@@ -704,7 +704,7 @@
 #define ext ".dylib"
 #else
 #include <dlfcn.h>
-#define ext ".so"
+#define ext ".so." xstr(X265_BUILD)
 #endif
 #if defined(__GNUC__) && __GNUC__ >= 8
 #pragma GCC diagnostic ignored "-Wcast-function-type"
