diff -Naur x265_4.0.old/source/CMakeLists.txt x265_4.0/source/CMakeLists.txt
--- x265_4.0.old/source/CMakeLists.txt	2024-10-01 10:53:48.836239436 +0200
+++ x265_4.0/source/CMakeLists.txt	2024-10-01 10:53:48.925240131 +0200
@@ -838,7 +838,15 @@
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
-        set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        if(HIGH_BIT_DEPTH)
+            if(MAIN12)
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main12)
+            else()
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main10)
+            endif()
+        else()
+            set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        endif(HIGH_BIT_DEPTH)
     endif()
     if(UNIX)
         set_target_properties(x265-shared PROPERTIES VERSION ${X265_BUILD})
diff -Naur x265_4.0.old/source/encoder/api.cpp x265_4.0/source/encoder/api.cpp
--- x265_4.0.old/source/encoder/api.cpp	2024-10-01 10:53:48.868239686 +0200
+++ x265_4.0/source/encoder/api.cpp	2024-10-01 10:53:48.926240139 +0200
@@ -1102,7 +1102,7 @@
 #define ext ".dylib"
 #else
 #include <dlfcn.h>
-#define ext ".so"
+#define ext ".so." xstr(X265_BUILD)
 #endif
 #if defined(__GNUC__) && __GNUC__ >= 8
 #pragma GCC diagnostic ignored "-Wcast-function-type"
