diff -Naur x265_3.4.old/source/CMakeLists.txt x265_3.4/source/CMakeLists.txt
--- x265_3.4.old/source/CMakeLists.txt	2020-06-23 21:23:30.696701498 +0200
+++ x265_3.4/source/CMakeLists.txt	2020-06-23 21:23:40.196025308 +0200
@@ -220,7 +220,7 @@
         add_definitions(-std=gnu++98)
     endif()
     if(ENABLE_PIC)
-         add_definitions(-fPIC)
+         add_definitions(-fPIC -DPIC)
     endif(ENABLE_PIC)
     if(NATIVE_BUILD)
         if(INTEL_CXX)
