diff -Naur x265_3.0.old/source/CMakeLists.txt x265_3.0/source/CMakeLists.txt
--- x265_3.0.old/source/CMakeLists.txt	2019-02-26 22:29:35.511067510 +0100
+++ x265_3.0/source/CMakeLists.txt	2019-02-26 22:30:07.411655550 +0100
@@ -212,7 +212,7 @@
         add_definitions(-std=gnu++98)
     endif()
     if(ENABLE_PIC)
-         add_definitions(-fPIC)
+         add_definitions(-fPIC -DPIC)
     endif(ENABLE_PIC)
     if(NATIVE_BUILD)
         if(INTEL_CXX)
