diff -Naur multicoreware-x265_git-931178347b3f.old/source/CMakeLists.txt multicoreware-x265_git-931178347b3f/source/CMakeLists.txt
--- multicoreware-x265_git-931178347b3f.old/source/CMakeLists.txt	2022-09-16 09:59:59.187261534 +0200
+++ multicoreware-x265_git-931178347b3f/source/CMakeLists.txt	2022-09-16 10:03:34.625384755 +0200
@@ -247,26 +247,6 @@
             endif()
         endif()
     endif()
-    if(ARM AND CROSS_COMPILE_ARM)
-        message(STATUS "cross compile arm")
-		set(ARM_ARGS -march=armv6 -mfloat-abi=soft -mfpu=vfp -marm -fPIC)
-    elseif(ARM)
-        find_package(Neon)
-        if(CPU_HAS_NEON)
-            set(ARM_ARGS -mcpu=native -mfloat-abi=hard -mfpu=neon -marm -fPIC)
-            add_definitions(-DHAVE_NEON)
-        else()
-            set(ARM_ARGS -mcpu=native -mfloat-abi=hard -mfpu=vfp -marm)
-        endif()
-    endif()
-	if(ARM64 OR CROSS_COMPILE_ARM64)
-	    set(ARM_ARGS -fPIC -flax-vector-conversions)
-        find_package(Neon)
-        if(CPU_HAS_NEON)
-            add_definitions(-DHAVE_NEON)
-        endif()
-    endif()
-    add_definitions(${ARM_ARGS})
     if(FPROFILE_GENERATE)
         if(INTEL_CXX)
             add_definitions(-prof-gen -prof-dir="${CMAKE_CURRENT_BINARY_DIR}")
diff -Naur multicoreware-x265_git-931178347b3f.old/source/dynamicHDR10/CMakeLists.txt multicoreware-x265_git-931178347b3f/source/dynamicHDR10/CMakeLists.txt
--- multicoreware-x265_git-931178347b3f.old/source/dynamicHDR10/CMakeLists.txt	2022-09-16 09:56:11.534906598 +0200
+++ multicoreware-x265_git-931178347b3f/source/dynamicHDR10/CMakeLists.txt	2022-09-16 10:03:48.721656028 +0200
@@ -42,18 +42,6 @@
             endif()
         endif()
     endif()
-    if(ARM AND CROSS_COMPILE_ARM)
-        set(ARM_ARGS -march=armv6 -mfloat-abi=soft -mfpu=vfp -marm -fPIC)
-    elseif(ARM)
-        find_package(Neon)
-        if(CPU_HAS_NEON)
-            set(ARM_ARGS -mcpu=native -mfloat-abi=hard -mfpu=neon -marm -fPIC)
-            add_definitions(-DHAVE_NEON)
-        else()
-            set(ARM_ARGS -mcpu=native -mfloat-abi=hard -mfpu=vfp -marm)
-        endif()
-    endif()
-    add_definitions(${ARM_ARGS})
     if(FPROFILE_GENERATE)
         if(INTEL_CXX)
             add_definitions(-prof-gen -prof-dir="${CMAKE_CURRENT_BINARY_DIR}")
@@ -150,4 +138,4 @@
 option(ENABLE_SHARED "Build shared library" OFF)
 
 install(FILES hdr10plus.h DESTINATION include)
-endif()
\ No newline at end of file
+endif()
