diff -Naur multicoreware-x265_git-34532bda12a3.old/source/CMakeLists.txt multicoreware-x265_git-34532bda12a3/source/CMakeLists.txt
--- multicoreware-x265_git-34532bda12a3.old/source/CMakeLists.txt	2023-05-31 16:17:39.525067979 +0200
+++ multicoreware-x265_git-34532bda12a3/source/CMakeLists.txt	2023-05-31 16:16:47.467272196 +0200
@@ -687,7 +687,15 @@
     if(MSVC)
         set_target_properties(x265-shared PROPERTIES OUTPUT_NAME libx265)
     else()
-        set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        if(HIGH_BIT_DEPTH)
+            if(MAIN12)
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main12)
+            else()
+                set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265_main10)
+            endif()
+        else()
+            set_target_properties(x265-shared PROPERTIES OUTPUT_NAME x265)
+        endif(HIGH_BIT_DEPTH)
     endif()
     if(UNIX)
         set_target_properties(x265-shared PROPERTIES VERSION ${X265_BUILD})
diff -Naur multicoreware-x265_git-34532bda12a3.old/source/encoder/api.cpp multicoreware-x265_git-34532bda12a3/source/encoder/api.cpp
--- multicoreware-x265_git-34532bda12a3.old/source/encoder/api.cpp	2023-05-31 16:16:07.866666833 +0200
+++ multicoreware-x265_git-34532bda12a3/source/encoder/api.cpp	2023-05-31 16:16:47.468272212 +0200
@@ -1097,7 +1097,7 @@
 #define ext ".dylib"
 #else
 #include <dlfcn.h>
-#define ext ".so"
+#define ext ".so." xstr(X265_BUILD)
 #endif
 #if defined(__GNUC__) && __GNUC__ >= 8
 #pragma GCC diagnostic ignored "-Wcast-function-type"
