diff -Naur x265_2.7.old/source/CMakeLists.txt x265_2.7/source/CMakeLists.txt
--- x265_2.7.old/source/CMakeLists.txt	2018-04-24 21:40:17.644790954 +0200
+++ x265_2.7/source/CMakeLists.txt	2018-04-24 21:40:52.199426580 +0200
@@ -204,7 +204,7 @@
         add_definitions(-std=gnu++98)
     endif()
     if(ENABLE_PIC)
-         add_definitions(-fPIC)
+         add_definitions(-fPIC -DPIC)
     endif(ENABLE_PIC)
     if(NATIVE_BUILD)
         if(INTEL_CXX)
