diff -Naur x265_4.0.old/source/test/testharness.h x265_4.0/source/test/testharness.h
--- x265_4.0.old/source/test/testharness.h	2024-10-01 10:53:48.878239764 +0200
+++ x265_4.0/source/test/testharness.h	2024-10-01 10:53:48.881239788 +0200
@@ -71,7 +71,7 @@
 #include <intrin.h>
 #elif (!defined(__APPLE__) && (defined (__GNUC__) && (defined(__x86_64__) || defined(__i386__))))
 #include <x86intrin.h>
-#elif ( !defined(__APPLE__) && defined (__GNUC__) && defined(__ARM_NEON__))
+#elif ( !defined(__APPLE__) && defined(__ARM_NEON__))
 #include <arm_neon.h>
 #else
 /* fallback for older GCC/MinGW */
